common/lib/capa/capa/templates/javascriptinput.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
This does seem like a simple one to add, but I would clean up the new errors, which probably means deleting the `|h`'s that are no longer needed.